spring-core/src/test/java/org/springframework/core/annotation/AnnotationTypeMappingsTests.java

```diff
@@ -474,6 +474,16 @@ void isEquivalentToDefaultValueWhenNotMatchingReturnsFalse() {
 		return result;
 	}
 
+	@Test
+	void forAnnotationTypeWhenRepeatableMetaAnnotationFilterd() {
+		AnnotationTypeMappings mappings = AnnotationTypeMappings.forAnnotationType(WithRepeatedMetaAnnotations.class,
+		annotationType ->
+				ObjectUtils.nullSafeEquals(annotationType, Repeating.class.getName()));
```

Comment:
You omitted the import for `ObjectUtils`, but don't worry about it. I'll add it before merging.

Reply:
For future reference, I was able to simplify the annotation filter in that test method as follows.

```java
@Test
void forAnnotationTypeWhenRepeatableMetaAnnotationIsFiltered() {
	AnnotationTypeMappings mappings = AnnotationTypeMappings.forAnnotationType(WithRepeatedMetaAnnotations.class,
			Repeating.class.getName()::equals);
	assertThat(getAll(mappings)).flatExtracting(AnnotationTypeMapping::getAnnotationType)
			.containsExactly(WithRepeatedMetaAnnotations.class);
}
```

However, I won't be merging that change due to reasons I'll explain later.